pandas/tseries/holiday.py

```diff
@@ -559,6 +559,7 @@ class USFederalHolidayCalendar(AbstractHolidayCalendar):
         USMartinLutherKingJr,
         USPresidentsDay,
         USMemorialDay,
+        Holiday("Juneteenth", month=6, day=19, observance=nearest_workday),
```

Comment:
Should probably be `"Juneteenth National Independence Day"` https://www.opm.gov/policy-data-oversight/pay-leave/federal-holidays/#url=2021